ddl/column.go

```diff
@@ -37,7 +37,7 @@ func (d *ddl) adjustColumnOffset(columns []*model.ColumnInfo, indices []*model.I
 		for i := 0; i < len(columns); i++ {
 			colOffset := columns[i].Offset
 			if colOffset != i {
-				if colOffset != lastColOffset { // column before the add action,
+				if colOffset < lastColOffset { // column before the add action,
```

Comment:
Some column with (first|after) position property was added into old columns.  All old columns's offset  may be changed after one column appended into head.
c1 c2 c3 =》 c4 c1 c5 c2 c3
1  2   3          4   1   5  2   3